corehq/apps/domainsync/management/commands/copy_domain.py

```diff
@@ -88,7 +90,7 @@ def handle(self, *args, **options):
         simulate = options['simulate']
         exclude_attachments = options['exclude_attachments']
 
-        since = datetime.strptime(options['since'], '%Y-%m-%d').isoformat() if options['since'] else None
+        since = json_format_date(iso_string_to_date(options['since'])) if options['since'] else None
```

Comment:
This is technically different because it returns '2014-04-01' instead of that plus 'T00:00:00Z', but that's ok because this is just used in a startkey for couchdb, and seems cleaner anyway